spec/system/planning_applications/review/pre_commencement_conditions_spec.rb

```diff
@@ -0,0 +1,173 @@
+
+        expect(page).to have_content(standard_condition.title)
+        expect(page).to have_content(standard_condition.text)
+        expect(page).to have_content(standard_condition.reason)
+
+        expect(page).to have_content(other_condition.text)
+        expect(page).to have_content(other_condition.reason)
+
+        choose "Accept"
+
+        click_button "Save and mark as complete"
+
+        expect(page).to have_content("Review conditions successfully updated")
+
+        expect(page).to have_list_item_for(
+          "Review pre-commencement conditions",
+          with: "Completed"
+        )
+
+        condition_set = planning_application.pre_commencement_condition_set
+        expect(condition_set.current_review.action).to eq "accepted"
+        expect(condition_set.current_review.review_status).to eq "review_complete"
+      end
+
+      it "I can edit to accept the planning officer's decision" do
+        expect(page).to have_list_item_for(
+          "Review pre-commencement conditions",
+          with: "Not started"
+        )
+
+        click_link "Review pre-commencement conditions"
+
+        choose "Edit to accept"
+
+        within("#condition-set-conditions-attributes-1--destroy-conditional") do
+          fill_in "Reason", with: "This is different reason"
+        end
+
+        click_button "Save and mark as complete"
+
+        expect(page).to have_content("Review conditions successfully updated")
+
+        expect(page).to have_list_item_for(
+          "Review pre-commencement conditions",
+          with: "Completed"
+        )
+
+        condition_set = planning_application.pre_commencement_condition_set
+        expect(condition_set.current_review.action).to eq "edited_and_accepted"
+        expect(condition_set.conditions.last.reason).to eq "This is different reason"
+        expect(condition_set.current_review.review_status).to eq "review_complete"
+      end
+
+      it "I can return to officer with comment" do
+        expect(page).to have_list_item_for(
+          "Review pre-commencement conditions",
+          with: "Not started"
+        )
+
+        click_link "Review pre-commencement conditions"
+
+        choose "Return to officer with comment"
+
+        fill_in "Comment", with: "I don't think you've assessed conditions correctly"
+
+        click_button "Save and mark as complete"
+
+        expect(page).to have_content("Review conditions successfully updated")
+
+        expect(page).to have_list_item_for(
+          "Review pre-commencement conditions",
+          with: "To be reviewed"
+        )
+
+        condition_set = planning_application.pre_commencement_condition_set
+        expect(condition_set.current_review.action).to eq "rejected"
+        expect(condition_set.current_review.comment).to eq "I don't think you've assessed conditions correctly"
+        expect(condition_set.current_review.status).to eq "to_be_reviewed"
+
+        # sign_out(reviewer)
```

Comment:
Should this all be commented out?

Reply:
No. 🤦🏻 I've just uncommented and realised that the reviewer comment doesn't show when returned to the assessor.